### PR TITLE
Issue-#163-Resolved

### PR DESCRIPTION
**Pull request by pragyamishra56**
1. **Issue Number:** #163
2. **Description:** Modernize JavaScript usage in css-examples.
3. **Resolution:** Replaced 'var' with 'const/let'.
4. **Outcome:** Enhanced code readability and adherence to modern standards.

### CODE REVIEW

File: editable-samples-2/js/prism.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
...anguage:t,parent:r};if("comment"==l.type&&(l.attributes.spellcheck="true"),e.alias){var i="Array"===n.util.type(e.alias)?e.alias:[e.alias];Array.prototype.push.apply(l.classes,i)}n.hooks.run("wrap",l);var o=Object.keys(l.attributes).map(function(e){return e+'="'+(l.attributes[e]||"").replace(/"/g,"&quot;")+'"'}).join(" ");return"<"+l.tag+' class="'+l.classes.join(" ")+'"'+(o?" "+o:"")+">"+l.content+"</"+l.tag+">"},!_self.document)return _self.addEventListener?(_self.addEventListener("message",function(e){var t=JSON.parse(e.data),a=t.language,r=t.code,l=t.immediateClose;_self.postMessage(n.highlight(r,n.languages[a],a)),l&&_self.close()},!1),_self.Prism):_self.Prism;var r=document.currentScript||[].slice.call(document.getElementsByTagName("script")).pop();return r&&(n.filename=r.src,!document.addEventListener||n.manual||r.hasAttribute("data-manual")||("loading"!==document.readyState?window.requestAnimationFrame?window.requestAnimationFrame(n.highlightAll):window.setTimeout(n.highlightAll,16):document.addEventListener("DOMContentLoaded",n.highlightAll))),_self.Prism}();"undefined"!=typeof module&&module.exports&&(module.exports=Prism),"undefined"!=typeof global&&(global.Prism=Prism);
```
Comment (yin1999) [inline editable-samples-2/js/prism.js:2]: I believe this file should not be touched. This is a lib download from https://prismjs.com/download.html

Comment (pragyamishra56) [reply editable-samples-2/js/prism.js:2]: Yes sir

**Comment by teoli2003** [comment]
The problem here is that you have done a find & replace.

The goal is not to replace all `var` by `let`, but to use `const` where possible too. Also, there are likely cases where `var` is correct, or where more changes are needed (as `var` behaviour may be different, especially wrt scope)

Can you start with 1 file or 1 group of files?

**Comment by pragyamishra56** [comment]
@teoli2003 Sir I appreciate the clarification. I agree that a more thorough approach is needed. I'll work on breaking down the changes into separate PRs for a more comprehensive review. Thanks for guiding me through this process.

**Pull request closed without merging**